### ISSUE TITLE
FPP 7.3.1 "FPPD Daemon is not running"

### ISSUE THREAD

**Issue by GitHubGoody** [FalconChristmas/fpp#1640]
**FPP 7.3.1 (fresh image) running on RPi Zero W on 5V 2.5A power supply**: 

**Describe the bug and Steps to reproduce**: 
A clear and concise description of what the bug is and the steps to be taken to reproduce. 
1. Install fresh v7.3.1 image from Raspberry Pi Imager
2. Complete initial configuration, file storage expansion, and network configuration (including enabling Samba for file upload)
3. Upload 3 .mp4 videos via Samba
4. Create a playlist
5. Play the playlist (in my case using default HDMI output to a projector)
6. Wait 10-20 minutes
7. See error message
8. Try to do anything other than Reboot the entire RPi
9. Observe that the UI indicates successful completion of most functions (e.g. Start FPPD, etc.), but does not actually complete them successfully
10. Reboot the RPi
11. Start back at 5
12. Submit Bug report

![IMG_4442](https://github.com/FalconChristmas/fpp/assets/46235745/3eee1db8-05e2-40f7-a400-6cd46f6f240c)
![IMG_4443](https://github.com/FalconChristmas/fpp/assets/46235745/c0d425ca-e9a4-4705-8817-0cd8abe6d140)

**Expected behavior**: 
I expected FPP to not stop running

**Additional context**:
N/A
[FPP_Logs_20231110.1312.zip](https://github.com/FalconChristmas/fpp/files/13319633/FPP_Logs_20231110.1312.zip)

**Additional Attachments**
Log files attached.

**Comment by darylc** [comment]
2023-11-10 07:44:43.812 (12732) [MediaOut] /opt/fpp/src/mediaoutput/VLCOut.cpp:140: MMAL error 2 "ENOSPC" 

This might indicate your pi is running out of memory perhaps, pi zero w has very little memory.  I'd suggest trying a very small video file or a pi with more memory and see if you have the same problem?

**Comment by derwin12** [comment]
May mean nothing but Looking in syslog..
Nov 10 12:44:43 FPP-Garage fppd[1606]: mmal: mmal_port_event_get: vc.ril.video_decode(1:0) port 0xaa514750, no event buffer left for ERRO
Nov 10 12:44:43 FPP-Garage fppd[1606]: mmalipc: mmal_vc_handle_event_msg: no event buffer available to receive event (2)

This talks about " GPU memory size was set to the default value of 64MB. Setting this to 256MB" in Raspberry menu>Settings>Raspberry Pi Configuration>Performance fixed it. 
Not sure that memory setting is exposed in FPP.

https://forums.raspberrypi.com/viewtopic.php?t=313554

**Comment by dkulp** [comment]
The PiZeroW really does not have enough memory to handle videos, particularly any high def videos.     You may be able to use it if you re-encode the videos down to 720p or 480p, but anything above that will definitely cause problems.  Increasing the gpu_memory is a temporary option, but then you may not be able to update FPP as there wouldn't be enough regular memory left to be able to compile/link FPP.     For videos, we REALLY recommend a full Pi3B+ or 4B.

**Comment by GitHubGoody** [comment]
@darylc and @derwin12
Thank you. I looked into this and was able to change the GPU memory to 256MB (from 128MB). For anyone stumbling across this in the future, I did not find this setting in FPP. I ssh'd into the RPi (default username/password are fpp/falcon) then sudo raspi-config > Performance > GPU Memory > Enter a value > OK > Finish > reboot

But, alas, the RPi still froze.

So, @dkulp Thank you as well. It definitely took reducing the file size (I'm currently using 854x480) to get the RPi to not crash. I think this is the native resolution of my cheapo projector (Goodee YG-500) which just downsamples HD video anyways, so nothing should be lost...except for the instability hopefully! I've been running a playlist for a few hourse without any crashes so far!
